decidim-core/app/packs/stylesheets/decidim/_redesigned_layout.scss

```diff
@@ -34,11 +34,11 @@
   @apply md:grid grid-cols-12 container grow;
 
   &__aside {
-    @apply col-span-3 md:pr-16 py-6 md:py-12 gap-6 md:gap-12 flex flex-col justify-between items-start md:justify-start before:content-[''] before:absolute before:top-0 before:left-0 before:h-full before:w-1/2 before:-z-10 md:before:bg-background;
+    @apply col-span-4 md:pr-16 py-6 md:py-12 gap-6 md:gap-12 flex flex-col justify-between items-start md:justify-start before:content-[''] before:absolute before:top-0 before:left-0 before:h-full before:w-1/2 before:-z-10 md:before:bg-background;
```

Comment:
Be careful, this file is widely shared along the app, and here you just getting wider the grey column to any kind of device size.

You need to provide breakpoint classes to point only tablet devices. That's something like `col-span-4 lg:col-span-3` for instance

Reply:
Hi @Crashillo could expand a bit more by what you mean "breakpoint" classes? Since this class is only pointing to part of the container on the page ```&__aside```I would assume it would not affect other files shared on the app? 

In short you would prefer the line to look like so: ```@apply col-span-4 lg:col-span-3 md:pr-16 py-6 md:py-12 gap-6 md:gap-12 flex flex-col justify-between items-start md:justify-start before:content-[''] before:absolute before:top-0 before:left-0 before:h-full before:w-1/2 before:-z-10 md:before:bg-background;``` 

Keen to get your thoughts.

Reply:
`&__aside` is used within the `layout-2col` css class, e.g. https://decidim-redesign.populate.tools/meetings. As you can see, this two columns composition is a quite common pattern all along the site.

By _breakpoint classes_, I should rather have said responsive classes 😄. The point is you need to make it work all of them properly, no matter which resolution, but taking care of the original design. The change you suggested will break this one, that's why you have to apply the responsive classes accordingly.

See further info about [tailwind responsive design](https://tailwindcss.com/docs/responsive-design)

Reply:
Thanks @Crashillo. You mention my suggestion would break the code. Are you referring to my original commit: 9d5bc2fe525a9c28dd7031fbe8d80b6d4048c8e9 or my suggestion in the last comment in our thread?

Reply:
Yep, the original one, https://github.com/decidim/decidim/commit/9d5bc2fe525a9c28dd7031fbe8d80b6d4048c8e9